Subcategories (#861)
UniMath/agda-unimath: src/foundation/symmetric-difference.lagda.md

## Idea

> The **symmetric difference** of two [subtypes](foundation-core.subtypes.md) `A`
> and `B` is the subtypes that contains the elements that are either in `A` or in
> `B` but not in both.

Reviewer: ```suggestion
The **symmetric difference** of two [subtypes](foundation-core.subtypes.md) `A`
and `B` is the subtype that contains the elements that are either in `A` or in
`B` but not in both.
```

### The coproduct of two decidable subtypes is equivalent to their symmetric difference plus two times their intersection

> This is also known as the _inclusion-exclusion principle_.

Reviewer: This is indeed closely related to the inclusion-exclusion principle, but I'm not sure if it is exactly that. I know it as the number of elements in the union of A and B is the number of elements in A plus the number of elements in B minus the number of elements in their intersection. So the way I know it is about cardinalities.

Perhaps we could say:
```suggestion
This is closely related to the _inclusion-exclusion principle_.
```
